ferrocene/doc/user-manual/src/targets/thumbv7em-none-eabihf.rst

```diff
@@ -0,0 +1,33 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. _thumbv7em-none-eabihf:
+
+:target:`thumbv7em-none-eabihf`
+==============================
+
+The ``thumbv7em-none-eabihf`` Ferrocene target provides support for
+bare-metal ARMv7E-M processors using the
+`T32 ISA <https://developer.arm.com/Architectures/T32%20Instruction%20Set%20Architecture>`_
+with hard floating point.
```

Comment:
The distinction between the targets is around the ABI, not the use of floating point instructions, and I think this line doesn't make that clear. 

Unsure if it's better to refer to or copy the wording at https://doc.rust-lang.org/nightly/rustc/platform-support/arm-none-eabi.html and https://doc.rust-lang.org/nightly/rustc/platform-support/thumbv7em-none-eabi.html

Reply:
I think we can clarify a bit and refer to that document.